contracts/prize-strategy/multiple-winners/MultipleWinners.sol

```diff
@@ -62,7 +62,7 @@ contract MultipleWinners is PeriodicPrizeStrategy {
     return true;
   }
 
-  function setCarryBacklist(bool _carry) external onlyOwner requireAwardNotInProgress returns (bool) {
+  function setCarryBlocklist(bool _carry) external onlyOwner requireAwardNotInProgress returns (bool) {
```

Comment:
`setCarryOverBlocklist` to match the variable.

Reply:
**This function should match the public getter**

The public getter is: 

```solidity
bool public carryOverBlocklist;
```

So either:

- this should be `setCarryOverBlocklist`
- the getter is changed to match